docs/reference/release-notes/6.4.asciidoc

```diff
@@ -56,6 +63,13 @@ Circuit Breakers::
 Java High Level REST Client::
 * HLRC: Fixing bug when getting a missing pipeline {pull}34286[#34286] (issue: {issue}34119[#34119])
 
+Machine learning::
+* Fixes the cause of hard_limit memory errors for jobs with bucket spans greater 
```

Comment:
Maybe put `hard_limit` in backticks?  (Maybe there is a precedent for not doing this though, in which case don't.)

Reply:
Thanks @droberts195 , I made that change